apps/userdashboard/assets/js/a4_candy_userdashboard/AiReportExplanation.jsx

```diff
@@ -11,31 +12,39 @@ const translated = {
   readLess: django.gettext('Read less')
 }
 
-export const AIReportExplanation = ({ AIReport }) => {
+export const AiReportExplanation = ({ AIReport, notificationPk }) => {
   const [isExpanded, setIsExpanded] = useState()
 
   const toggleExpand = () => {
     setIsExpanded(!isExpanded)
   }
 
   return (
-    <li className="alert alert--danger d-flex text-start mb-3">
-      <i
-        className="fas fa-exclamation-circle text-danger pt-1 pe-2"
-        aria-hidden="True"
-      />
-      <p>
-        {translated.intro + ' '}
-        {isExpanded && AIReport + ' '}
-        <button
-          className="btn--link text-danger"
-          type="button"
-          onClick={toggleExpand}
-          aria-label={isExpanded ? translated.ariaReadLess : translated.ariaReadMore}
-        >
-          {isExpanded ? translated.readLess : translated.readMore}
-        </button>
-      </p>
+    <li className="alert alert--danger mb-3">
+      <div className="d-flex text-start mb-4">
+        <i
+          className="fas fa-exclamation-circle text-danger pt-1 pe-2"
+          aria-hidden="True"
+        />
+        <p>
+          {translated.intro + ' '}
+          {isExpanded && AIReport + ' '}
+          <button
+            className="btn--link text-danger"
+            type="button"
+            onClick={toggleExpand}
+            aria-label={isExpanded ? translated.ariaReadLess : translated.ariaReadMore}
+          >
+            {isExpanded ? translated.readLess : translated.readMore}
+          </button>
+        </p>
+      </div>
+      <div className="d-flex text-start">
+        <SwitchButton
+          id={notificationPk}
+          switchLabel={translated.readLess}
```

Comment:
did you leave this as a placeholder on purpose or should it be `Hide info from users` ?